scripts/benchmarks_local.py

```diff
@@ -76,36 +75,38 @@ def get_os_short_name(os_name: str):
     else:
         raise ValueError(f"Unknown OS {os_name}")
 
-def is_running_as_admin(parsed_args: Namespace):
+def is_running_as_admin(parsed_args: Namespace) -> bool:
     if is_windows(parsed_args):
+        import ctypes
         return ctypes.windll.shell32.IsUserAnAdmin()
     else:
-        return os.getuid() == 0
+        return os.getuid() == 0 # type: ignore os.getuid() returns an int on Unix-like systems
```

Comment:
It's fine that it returns int. You're comparing it with 0, which is also int. Or am I missing something?

Reply:
Pylance (for me) displays `Type of "getuid" is unknown`. Clarified this in the latest push.